rules/config

```diff
@@ -145,3 +145,10 @@ KUBERNETES_VERSION = 1.18.0
 K8s_GCR_IO_PAUSE_VERSION = 3.2
 K8s_CNI_CALICO_VERSION = 3.12.1
 
+# SONIC_ENABLE_IMAGE_SIGNATURE - enable image signature
+SONIC_ENABLE_IMAGE_SIGNATURE = y
+
+# SONIC_CETIFICATE_PATH - the certificate path
+# The expected files in the folder: signing.key, signing.cert, ca.cert
+# If the parameter SONIC_CETIFICATE_PATH is no specified, then the test self-signed certificate will be generated
```

Comment:
I mean we could define the config the same way as your CLI.

Developer could config `SIGNING_KEY` `SIGNING_CERT` `CA_CERT`, and no need to add `SONIC_ENABLE_IMAGE_SIGNATURE`.

Think about the use case of non-test sign, the current implementation assumes the filenames, which is not convenient.

Reply:
Set the following value in rules/config, right? How to specify a CA_CERT value for test?
# SIGNING_KEY = 
# SIGNING_CERT = 
CA_CERT = ?

Reply:
Ether way is ok.
It is difficult to understand what is the relative paths relative to. Maybe add some comment?

Reply:
I will add the ENABLE_IMAGE_SIGNATURE = y, since the generated ca.cert should be copied to the target path to install, but the target path is set in slave.mk, hard code target for CA_CERT seems not good enough.